data/ships.txt

```diff
@@ -867,6 +867,45 @@ ship "Combat Drone"
 
 
 
+ship "Heavy Combat Drone"
```

Comment:
Mission specific ships should be next to their missions. See the timer ship or hallucination ship.